smarts/sstudio/genhistories.py

```diff
@@ -229,8 +233,19 @@ def __init__(self, dataset_spec, output):
             "position_y": "x" if self._swap_xy else "y",
         }
 
+    def check_dataset_spec(self, dataset_spec: Dict[str, Any]):
+        super().check_dataset_spec(dataset_spec)
+        hiw = dataset_spec.get("heading_inference_window", 2)
+        if hiw != 2:
+            # Adding support fot this would require changing the rows() generator
+            # (since we're not using Pandas here like we are for NGSIM).
+            # So wait until if/when users request it...
+            raise ValueError(
+                "heading_inference_window not yet supported for Interaction datasets."
+            )
```

Comment:
Do we want to raise an Error here? Shouldn't we just warn the user this is not supported for now?

Reply:
I think I'm going to keep it a fatal error for now (for the reason I wrote in your other comment).